### PR TITLE
Add Explorer HAT to reflash section

### PR DESCRIPTION
**Pull request by juehv**

### CODE REVIEW

File: docs/docs/Resources/troubleshooting.md [hunk 1]
```diff
@@ -216,25 +216,36 @@ Below is correct definition
 Full error is usually: 
 `Could not get subg_rfspy state or version. Have you got the right port/device and radio_type? (ccprog)`
 
-Basic steps using an Intel Edison with Explorer Board, checking with `killall -g oref0-pump-loop; openaps mmtune` to see if it is resolved yet:
-  * Make sure the Explorer board has not become loose and is sitting correctly on the Edison board
+Basic steps using an Intel Edison with Explorer Board or a Raspberry Pi with Explorer HAT:
+  * checking with `killall -g oref0-pump-loop; openaps mmtune` to see if it is resolved yet
```
Comment (scottleibrand) [inline docs/docs/Resources/troubleshooting.md]: We should also update this section to reflect the new Go command needed for dev.

**Comment by scottleibrand** [comment]
This looks good. Added on comment for a follow-up edit too.

**Comment by juehv** [comment]
But shouldn't we wait with the go information until 0.7 is released?

**Comment by danamlewis** [comment]
Approving as-is for now; please open another PR re: Go changes, and we can mark that as waiting for the 0.7.0 release before merging

**Comment by juehv** [comment]
Ok, will do that at the weekend :)

**Pull request merged**